libexec/scoop-alias.ps1

```diff
@@ -94,7 +94,7 @@ function list_aliases {
     }
 
     if (!$aliases.count) {
-        warn 'No aliases founds.'
+        Write-Host -ForegroundColor Yellow "No alias found."
```

Comment:
Why change this? The `warn` helper was made exactly for this purpose...

Reply:
I have a question: why is a warning message required where there is no alias found. For that I just think no alias doesn't affect the use of Scoop.

Reply:
It doesn't affect anything, but the user expects something to be shown when he types `scoop alias list`, and when no alias is there, a warning message is printed.

Reply:
Aha, I mean a plain message is just fine, why a warning one?

Reply:
We can change the `warn 'No aliases founds.'` to `info 'No aliases founds.'`.

Reply:
Yeah, that's almost exactly what I thought before: `info "No alias found."` I'll regroup all these changes and re-push.